Modules/_ctypes/callproc.c

```diff
@@ -1123,6 +1123,15 @@ PyObject *_ctypes_callproc(PPROC pProc,
                 goto cleanup; /* leaking ? */
             }
         }
+#if (defined(__x86_64__) && (defined(__MINGW64__) || defined(__CYGWIN__))) || \
+        defined(__aarch64__)
+        void *tmp;
```

Comment:
No offence taken :smile:

> I find variable declarations in an if statement pretty ugly

As far as I know, it's considered good practice for declarations to be limited to the scope of the blocks where they're used, as long as the version of C in use permits it. There's nothing special about an `if` block here - one would do the same for `for`, `while`, `do` etc.

> I won't bother making a separate commit for it

I wouldn't worry too much about that; commits are cheap and would normally be squashed on merge, anyway.